packages/frontend/core/src/components/blocksuite/block-suite-editor/blocksuite-editor-container.tsx

```diff
@@ -146,6 +146,9 @@ export const BlocksuiteEditorContainer = forwardRef<
       get page() {
```

Comment:
The `page` getter should be mark deprecated after https://github.com/toeverything/blocksuite/pull/6290

Reply:
It's just a compatibility layer here, so it doesn't matter